### PR TITLE
two quick changes (moderatism removed & redundant commented out stuff removed)

### PR DESCRIPTION
**Pull request by fleeingmoth**
we can always just look at vanilla for help making bookmark start dates

### CODE REVIEW

File: common/ideologies/00_ideologies.txt [hunk 1]
```diff
@@ -249,9 +252,6 @@ ideologies = {
 			neu_anarchism = {
 			}
 			
-			neu_moderatism = {
```
Comment (AndreasBrostrom) [inline common/ideologies/00_ideologies.txt:252]: i would perfere keeping this ideology.

File: localisation/atw_replace_l_english.yml [hunk 2]
```diff
@@ -56,9 +56,6 @@
  despotism_desc:0 "Despotism is a form of government in which a single person rules with absolute power."
  oligarchism:0 "Oligarchic"
  oligarchism_desc:0 "Oligarchy is a form of government in which a group of people rule with absolute power."
- moderatism:1 "Moderatism"
- moderatism_desc:1 "Moderatism is a form of government which avoids the extremes of left and right by taking a moderate position or course of action."
- centrism:1 "Centrism"
```
Comment (AndreasBrostrom) [inline localisation/atw_replace_l_english.yml:61]: You remove a used ideology here.

Name however now is `neu_centrism` or `dem_centrism`

File: localisation/atw_replace_l_english.yml [hunk 3]
```diff
@@ -56,9 +56,6 @@
  despotism_desc:0 "Despotism is a form of government in which a single person rules with absolute power."
  oligarchism:0 "Oligarchic"
  oligarchism_desc:0 "Oligarchy is a form of government in which a group of people rule with absolute power."
- moderatism:1 "Moderatism"
```
Comment (AndreasBrostrom) [inline localisation/atw_replace_l_english.yml:59]: i prefer keeping this ideology.
naming however is `dem_moderatism` or `neu_moderatism`

**Review by AndreasBrostrom — CHANGES_REQUESTED** [review]

File: common/ideologies/00_ideologies.txt (hunk 1 above)
Comment (fleeingmoth) [reply common/ideologies/00_ideologies.txt:252]: moderatism is just another word for centrism; no need to have both

**Comment by fleeingmoth** [comment]
how can i have multiple pull reqs

File: localisation/atw_replace_l_english.yml (hunk 2 above)
Comment (fleeingmoth) [reply localisation/atw_replace_l_english.yml:61]: see discussion above on moderatism

**Review by AndreasBrostrom — APPROVED** [review]

**Review by AndreasBrostrom — APPROVED** [review]

**Pull request merged**